Sources/Testcontainers/Docker/Models/ImageParams.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  ImageParams.swift
+//
+//
+//  Created by cristian on 3/09/24.
+//
+
+import Foundation
+
+public struct ImageParams {
```

Comment:
let's rename it to DockerImageName. Also, adding unit test will make sure it is parsing it correctly.

Reply:
Done. Much better